build_ngtf.py

```diff
@@ -351,9 +351,13 @@ def main():
             "-DENABLE_SPEECH_DEMO=FALSE",
             "-DCMAKE_INSTALL_RPATH=\"$ORIGIN\"",
             "-DCMAKE_INSTALL_PREFIX=" + os.path.join(artifacts_location,
-                                                     "openvino")
+                                                     "openvino"),
+            "-DENABLE_PYTHON=ON",
+            "-DPYTHON_EXECUTABLE=" + os.path.join(build_dir_abs,
```

Comment:
Instead of forcing Python3, can we get/read it from current venv?

Reply:
Our scripts assume python3 https://github.com/tensorflow/ngraph-bridge/blob/master/tools/build_utils.py#L125 so we can probably stick to it.